theme/style.css

```diff
@@ -69,5 +69,25 @@ img {
   text-shadow: none;
   border-bottom: 1px #000 dotted;
 }
+<<<<<<< HEAD
```

Comment:
`<<<<<` means that there was a merge conflict here! Make sure that you delete the remnants of said conflict.